plugins/extra/noaa.go

```diff
@@ -53,7 +119,7 @@ func tafCallback(b *seabird.Bot, m *irc.Message) {
 // look up the raw data. The first line is skipped, as that is generally the
 // date and the rest of the lines are joined together with a maximum of one
 // space between them.
-func NOAALookup(urlFormat, code string) (string, error) {
+func (p *noaaPlugin) NOAALookup(urlFormat, code string) (string, error) {
```

Comment:
Is there a reason this was changed? I think it was left this way because it can work separate from the plugin... if we're attaching it to the plugin it might make sense to make it private.